client/state/selectors/can-site-view-atomic-hosting.js

```diff
@@ -20,9 +21,17 @@ export default function canSiteViewAtomicHosting( state ) {
 
 	const siteId = getSelectedSiteId( state );
 
-	if ( ! isSiteOnAtomicPlan( state, siteId ) ) {
+	if ( ! canCurrentUser( state, siteId, 'manage_options' ) ) {
 		return false;
 	}
 
-	return canCurrentUser( state, siteId, 'manage_options' );
+	if ( false !== isJetpackSite( siteId ) || false !== isVipSite( siteId ) ) {
```

Comment:
Don't forget `state`! ✨This will cause Calypso to not load.

`isJetpackSite( state, siteId )`
`isVipSite( state, siteId )`

Reply:
Oops, lol. ccda6fc72c29b69e7a2ead779a2d1a8b760ad14a